pyramid_swagger/tween.py

```diff
@@ -542,10 +542,10 @@ def validate_response(response, validator_map):
 
 
 def prepare_body(response):
-    # content_type and charset must both be set to access response.text
-    if response.content_type is None or response.charset is None:
+    # content_type must be set to access response.text
+    if not response.content_type:
```

Comment:
I built up a concrete example in which it could fail.
With ``pyramid<1.8`` WebOb dependency is restricted to [``WebOb>=1.3.1``](https://github.com/Pylons/pyramid/blob/1.7.4/setup.py#L43), so if we're using an _old_ pyramid version like 1.5 as defined in ``tox`` we have no dependency restriction to ``WebOb>=1.7`` so it is possible to have ``pyramid==1.5,WebOb==1.3.1`` which now will raise ``AttributeError: You cannot access Response.text unless charset is set`` if charset is not defined (if won't happen with ``WebOB>=1.7.1``).

More reference on: [pyramid_swagger/test-fail-webob](https://github.com/macisamuele/pyramid_swagger/tree/test-fail-webob) and [travis build #39](https://travis-ci.org/macisamuele/pyramid_swagger/builds/208413993).

Said that I would enforce the usage of a newer ``WebOb`` dependency. An other possibility is to enforce the use of a newer ``pyramid`` version (which pins up ``WebOb``), but I think that it would be "too much".